patches/tModLoader/ReLogic/Content/Sources/IContentSource.cs.patch

```diff
@@ -54,6 +56,15 @@
 +		!Rejections.IsRejected(assetName) && GetExtension(assetName) != null;
 +
 +	// Added by TML.
-+	IEnumerable<string> GetAllAssetsStartingWith(string assetNameStart) =>
-+		EnumerateAssets().Where(s => s.StartsWith(assetNameStart));
++	[Obsolete, SuppressMessage("CodeQuality", "IDE0051:Remove unused private members")]
++	private IEnumerable<string> GetAllAssetsStartingWith(string assetNameStart)
++		=> GetAllAssetsStartingWith(assetNameStart, ignoreCase: false);
++
++	// Added by TML.
++	IEnumerable<string> GetAllAssetsStartingWith(string assetNameStart, bool ignoreCase = false)
++	{
++		var comparison = ignoreCase ? StringComparison.OrdinalIgnoreCase : StringComparison.Ordinal;
++
++		return EnumerateAssets().Where(s => s.StartsWith(assetNameStart, comparison));
```

Comment:
Is this really the correct solution? The entire asset system in vanilla is case insensitive. Should we just be using `.ToLowerInvariant()` everywhere instead?